es6/tests/base.js

```diff
@@ -916,3 +920,65 @@ describe("Serialization", function() {
 		JSON.stringify(doc);
 	});
 });
+
+describe("Docxtemplater v4 tests", function() {
+	it("should work with new constructor", function() {
+		let xmlDocuments;
+
+		const module = {
+			requiredAPIVersion: "3.0.0",
+			optionsTransformer(options, docxtemplater) {
+				const relsFiles = docxtemplater.zip
+					.file(/document.xml.rels/)
+					.map(file => file.name);
+				options.xmlFileNames = options.xmlFileNames.concat(relsFiles);
+				return options;
+			},
+			set(options) {
+				if (options.xmlDocuments) {
+					xmlDocuments = options.xmlDocuments;
+				}
+			},
+		};
+
+		const moduleV4Templater = {
+			requiredAPIVersion: "3.0.0",
+			optionsTransformer(options, docxtemplater) {
+				const relsFiles = docxtemplater.zip
+					.file(/document.xml.rels/)
+					.map(file => file.name);
+				options.xmlFileNames = options.xmlFileNames.concat(relsFiles);
+				return options;
+			},
+			set(options) {
+				if (options.xmlDocuments) {
+					xmlDocuments = options.xmlDocuments;
+				}
+			},
+		};
+
+		const doc = createDoc("tag-example.docx");
```

Comment:
I think you can use just one docxtemplater instance constructed with the v4 API.

Reply:
I am getting zip from the original doc. Won't the zip be effected if the transformation is not done by the module, on the `doc`?

Reply:
I'm not sure to understand your question here.

Reply:
It is changed now as the test have changed. 
What I was trying:
- make a doc from the old constructor, attach module, render it and then get the zip using 
`doc.getZip` to make another doc from the new constructor.
- Then validate the equality of two docs.

But It is changed now,  so no issues.

Reply:
ok